debian/rules

```diff
@@ -5,15 +5,13 @@
 
 override_dh_auto_build:
 	make CGO_CFLAGS="${CGO_CFLAGS}" CGO_LDFLAGS="${CGO_LDFLAGS}" CC=${CC} \
-	GOOS=linux GOARCH=arm GOARM=6 CGO_ENABLED=1 build
+	GOOS=linux CGO_ENABLED=1 build
 
-override_dh_auto_test:
-	true
+override_dh_auto_install:
+	# All install targets except install-bin and install-systemd
+	DESTDIR=debian/mender-client make install-bin install-identity-scripts install-inventory-scripts install-modules
```

Comment:
@kacf This is something I wanted to discuss. After mender-convert v2, I think we will be using the "make install" target only here for the deb package. So we might very well just modify it there removing the install-conf and install-systemd

Reply:
(This also made me realize that the comment is wrong, `install-bin` is there, what is missing is `install-conf`)

Reply:
I agree with `install-conf`, since we now have a replacement in `mender setup`, but I think `install-systemd` should stay. I think the `make install` stage should be modeled after what a user installing from source would use, not what our scripts use.

Reply:
Done in https://github.com/mendersoftware/mender/pull/443